src/base/static/libs/quill-image-resize.js

```diff
@@ -0,0 +1,273 @@
+/**
```

Comment:
I haven't been able to find it if it is, and in any case this plugin required a few modifications for it to work with our app, so we're self-hosting the modified version.